docs/maintenance/Pull_Requests.mdx

```diff
@@ -111,6 +108,19 @@ For preliminary reviews, be clear with what scale you're reviewing at: _"Reviewi
 For repeat reviews, be clear when it's something you missed earlier and/or there's new information.
 Don't apologize if the missed information was only made clear because of requested changes - this is part of the review process!
 
+### Approvals
+
+If the PR addresses a time critical task, such as a security fix or `main` branch build break, go ahead and squash merge it.
+
+Otherwise, upon completing your review, if the build is passing and you have no blockers, approve it on GitHub.
+Then:
+
+- If there isn't a `1 approval` label or existing approval, add the `1 approval` label
+- If there's already `1 approval` and/or it's been a week since the last request for changes, go ahead and squash merge
+  - For straightforward PRs that don't impact users, you can wait 3 days instead
```

Comment:
I guess we could also posit something along the lines of "no need to reset the 3 day timer for minor fixups from code reviews of otherwise approved code" or something, if that makes sense